lib/config.js

```diff
@@ -118,6 +119,9 @@ Config.DEFAULTS = {
       port: 80
     }
   },
+  logger: {
+    level: CONSTANTS.LOG_LEVEL_WARN
```

Comment:
If we're in the 'test' environment, we want to make sure that the log level is 0.